### ISSUE TITLE
RPM build failing check for packaged files on 1.2.17

### ISSUE THREAD

**Issue by jschwanz** [acassen/keepalived#164]
When running the `rpmbuild -ba keepalived.spec.in` build command, it fails with the following output:

```
Checking for unpackaged file(s): /usr/lib/rpm/check-files {buildroot}/keepalived-1.2.17-5.x86_64
error: Installed (but unpackaged) file(s) found:
   /usr/share/snmp/mibs/KEEPALIVED-MIB
   /usr/share/snmp/mibs/VRRP-MIB
    Installed (but unpackaged) file(s) found:
   /usr/share/snmp/mibs/KEEPALIVED-MIB
   /usr/share/snmp/mibs/VRRP-MIB
```

**Comment by thresheek** [comment]
Seems to be fixed with https://github.com/acassen/keepalived/commit/b2073ee95b9f54bb7e470d7bebc9bd443257eaa6

**Comment by jschwanz** [comment]
Still failing for me. Those two MIB files are still being copied into `$(DESTDIR)/usr/share/snmp/mibs/`.

**Comment by jschwanz** [comment]
Here's a few snipets from the log output:

```
...
+ ./configure --host=x86_64-redhat-linux-gnu --build=x86_64-redhat-linux-gnu --program-prefix= --prefix=/usr --exec-prefix=/usr --bindir=/usr/bin --sbindir=/usr/sbin --sysconfdir=/etc --datadir=/usr/share --includedir=/usr/include --libdir=/usr/lib64 --libexecdir=/usr/libexec --localstatedir=/var --sharedstatedir=/var/lib --mandir=/usr/share/man --infodir=/usr/share/info --with-kernel-dir=/lib/modules/2.6.32-431.5.1.el6/build
...
Keepalived configuration
------------------------
Keepalived version       : 1.2.18
Compiler                 : gcc
Compiler flags           : -O2 -g
Extra Lib                : -lssl -lcrypto -lcrypt 
Use IPVS Framework       : Yes
IPVS sync daemon support : Yes
IPVS use libnl           : No
fwmark socket support    : Yes
Use VRRP Framework       : Yes
Use VRRP VMAC            : Yes
SNMP support             : No
SHA1 support             : No
Use Debug flags          : No
...
install -m 644 ../doc/man/man1/genhash.1 /home/username/rpmbuild/BUILDROOT/keepalived-1.2.18-5.x86_64/usr/share/man/man1
make[1]: Leaving directory `/home/username/rpmbuild/BUILD/keepalived-1.2.18/genhash'
mkdir -p /home/username/rpmbuild/BUILDROOT/keepalived-1.2.18-5.x86_64/usr/share/snmp/mibs/
cp -f doc/VRRP-MIB /home/username/rpmbuild/BUILDROOT/keepalived-1.2.18-5.x86_64/usr/share/snmp/mibs/
cp -f doc/KEEPALIVED-MIB /home/username/rpmbuild/BUILDROOT/keepalived-1.2.18-5.x86_64/usr/share/snmp/mibs/
+ /bin/rm -rf /home/username/rpmbuild/BUILDROOT/keepalived-1.2.18-5.x86_64/etc/keepalived/samples/
+ /usr/lib/rpm/brp-compress
...
```

**Comment by thresheek** [comment]
Most likely your build still uses old .tar.gz.

**Comment by jschwanz** [comment]
Technically, yes. The RPM build process requires the existence of the .tar.gz. I don't know if the `Source0: http://www.keepalived.org/software/keepalived-%{version}.tar.gz` line in `keepalived.spec.in` is the cause of that.

**Comment by acassen** [comment]
fixed in 1.2.19
